### ISSUE TITLE
New submissions for Thu, 29 Feb 24

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#230]
# New submissions for Thu, 29 Feb 24
Auto update Star Formation & Molecular Cloud papers at about 2:30am UTC (10:30am Beijing time) every weekday.

阅读 `Usage.md`了解如何使用此repo实现个性化的Arxiv论文推送

See `Usage.md` for instructions on how to personalize the repo. 

Keyword list: ['star formation', 'star-forming', 'molecular cloud', 'interstellar medium', 'cloud', 'clump', 'core', 'filament', 'atomic gas', 'N-PDF']

Excluded: ['galaxies', 'galaxy cluster', ' AGN ']

#### scida: scalable analysis for scientific big data
 - **Authors:** Chris Byrohl, Dylan Nelson
 - **Subjects:** Instrumentation and Methods for Astrophysics (astro-ph.IM); Cosmology and Nongalactic Astrophysics (astro-ph.CO); Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2402.17818
 - **Pdf link:** https://arxiv.org/pdf/2402.17818
 - **Abstract**
 scida is a Python package for reading and analyzing large scientific data sets with support for various cosmological and galaxy formation simulations out-of-the-box. Data access is provided through a hierarchical dictionary-like data structure after a simple load() function. Using the dask library for scalable, parallel and out-of-core computation, all computation requests from a user session are first collected in a task graph. Arbitrary custom analysis, as well as all available dask (array) operations, can be performed. The subsequent computation is executed only upon request, on a target resource (e.g. a HPC cluster).
#### Missing Titanium in the Asymmetric Supernova Remnant W49B
 - **Authors:** Toshiki Sato, Makoto Sawada, Keiichi Maeda, John P. Hughes, Brian J. Williams
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2402.17957
 - **Pdf link:** https://arxiv.org/pdf/2402.17957
 - **Abstract**
 The progenitor of the W49B supernova remnant is still under debate. One of the candidates is a jet-driven core-collapse supernova. In such a highly asymmetric explosion, a strong $\alpha$-rich freezeout is expected in local high entropy regions, which should enrich elements synthesized by the capture of $\alpha$-particles such as $^{44}$Ti and $^{48}$Cr (decaying to $^{44}$Ca and $^{48}$Ti, respectively). In the present work, in order to infer the progenitor of the W49B remnant, we constrain the amount of stable Ti ($^{48}$Ti) synthesized, using the {\it Suzaku} observation. We found no firm evidence for the Ti line and set the upper limit of $M_{\rm Ti}/M_{\rm Fe} < 8.2 \times$ 10$^{-4}$ (99\% limit using Xspec) and $M_{\rm Ti}/M_{\rm Fe} < 1.9 \times$ 10$^{-3}$ (99\% limit using SPEX), and thus excluded almost all hypernova/jet-driven supernova models. Our results, as complemented by some previous studies, suggest that a Type Ia supernova from a near-$M_{\rm Ch}$ (Chandrasekhar mass) white dwarf is the most favorable candidate for the origin of W49B. Future observations with X-ray calorimeter missions, such as XRISM, will give us a stronger constraint on the progenitor.
#### From Simulations to Reality: Dark Energy Reconstruction with Simulated  SNIa data from the Vera C. Rubin Observatory
 - **Authors:** Ayan Mitra, Gómez-Vargas, Vasilios Zarikas
 - **Subjects:** Cosmology and Nongalactic Astrophysics (astro-ph.CO)
 - **Arxiv link:** https://arxiv.org/abs/2402.18124
 - **Pdf link:** https://arxiv.org/pdf/2402.18124
 - **Abstract**
 In this paper, we present an Artificial Neural Network (ANN) based reconstruction analysis of the Supernova Ia (SNIa) distance moduli ($\mu(z)$), and hence dark energy, using LSST simulated three-year SNIa data. Our ANN reconstruction architecture can model both the distance moduli and their corresponding error estimates. For this we employ astroANN and incorporate Monte Carlo dropout techniques to quantify uncertainties in our predictions. We tune our hyperparameters through advanced genetic algorithms, including elitism, utilizing the DEAP library. We compared the performance of the ANN based reconstruction with two theoretical descriptions of dark energy models, $\Lambda$CDM and Chevallier-Linder-Polarski (CPL). We perform a Bayesian analysis for these two theoretical models using the LSST simulations and also compare with observations from Pantheon and Pantheon+ SNIa real data. We show that our model-independent reconstruction using ANN is consistent with both of them. We assessed the performance using mean squared error (MSE) and showed that the ANN can produce distance estimates in better agreement with the LSST dataset than either $\Lambda$CDM or CPL, albeit very small. We included an additional residual analysis and a null test with $F$-scores to show that the reconstructed distances from the ANN model, are in excellent agreement with the $\Lambda$CDM or CPL model.
#### Studying The Effect of Radiation Pressure on Evolution of a Population  III Stellar Cluster
 - **Authors:** Sukalpa Kundu, Jayanta Dutta
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA); Cosmology and Nongalactic Astrophysics (astro-ph.CO)
 - **Arxiv link:** https://arxiv.org/abs/2402.18197
 - **Pdf link:** https://arxiv.org/pdf/2402.18197
 - **Abstract**
 Recent numerical simulations have shown that the unstable disk within the central regime of the primordial gas cloud fragments to form multiple protostars on several scales. Their evolution depends on the mass accretion phenomenon, interaction with the surrounding medium and radiative feedback respectively. In this work, we use a fast semi-analytical framework in order to model multiple protostars within a rotating cloud, where the mass accretion is estimated via a Bondi-Hoyle flow and the feedback process is approximated through radiation pressure. We observe that while some of the evolving protostars possibly grow massive ($\approx 1-24 M_{\odot}$) via accretion and mergers, a fraction of them ($\approx 20\%$) are likely to be ejected from the parent cloud with a mass corresponding to $M_{*} \lesssim 0.8 M_{\odot}$. These low-mass protostars may be considered as the potential candidates to enter the zero-age-main-sequence (ZAMS) phase and possibly survive till the present epoch.
#### Grey Two-moment Neutrino Transport: Comprehensive Tests and Improvements  for Supernova Simulations
 - **Authors:** Haakon Andresen, Evan P. O'Connor, Oliver Eggenberger Andersen, Sean M. Couch
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2402.18303
 - **Pdf link:** https://arxiv.org/pdf/2402.18303
 - **Abstract**
 In this work, we extended an energy-integrated neutrino transport method to facilitate efficient, yet precise, modeling of compact astrophysical objects. We focus particularly on core-collapse supernovae. We implemented the framework of Foucart et al. (2016) into FLASH and performed a detailed evaluation of its accuracy in core-collapse supernova simulations. Based on comparisons with results from simulations using energy-dependent neutrino transport, we incorporated several improvements to the original scheme. Our analysis shows that our grey neutrino transport method successfully reproduces key aspects from more complex energy-dependent transport across a variety of progenitors and equations of state. We find both qualitative and reasonable quantitative agreement with multi-group M1 transport simulations. However, the grey scheme tends to slightly favor shock revival. In terms of gravitational wave and neutrino signals, there is a good alignment with the energy-dependent transport, although we find 15-30 percent discrepancies in the average energy and luminosity of heavy-lepton neutrinos. Simulations using the grey transport are around four times faster than those using energy-dependent transport.
#### Discovery of an extended Horizontal Branch in the Large Magellanic Cloud  globular cluster NGC1835
 - **Authors:** Camilla Giusti, Mario Cadelano, Francesco R. Ferraro, Barbara Lanzoni, Cristina Pallanca, Maurizio Salaris, Emanuele Dalessandro, Enrico Vesperini, Alessio Mucciarelli
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2402.18389
 - **Pdf link:** https://arxiv.org/pdf/2402.18389
 - **Abstract**
 We present a high angular resolution multi-wavelength study of the massive globular cluster NGC 1835 in the Large Magellanic Cloud. Thanks to a combination of optical and near ultraviolet images acquired with the WFC3 on board the HST, we performed a detailed inspection of the stellar population in this stellar system adopting a ``UV-guided search'' to optimize the detection of relatively hot stars. This allowed us to discover a remarkably extended horizontal branch (HB), spanning more than 4.5 magnitudes in both magnitude and colour from the region redder than the instability strip, up to effective temperatures of 30,000 K, and including a large population of RR Lyrae (67 confirmed variables, and 52 new candidates). This is the first time that such a feature has been detected in an extra-Galactic cluster, demonstrating that the physical conditions responsible for the formation of extended HBs are ubiquitous. The acquired dataset has been also used to redetermine the cluster distance modulus, reddening, and absolute age, yielding $(m-M)_0=18.58$, $E(B-V)=0.08$, and $t=12.5$ Gyr, respectively.
#### Correction method applied to MC simulated LST images affected by clouds
 - **Authors:** Natalia Żywucka, Julian Sitarek, Dorota Sobczyńska, Mario Pecimotika, Dario Hrupec, Dijana Dominis Prester, Lovro Pavletić, Saša Mićanović
 - **Subjects:** Instrumentation and Methods for Astrophysics (astro-ph.IM); High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2402.18416
 - **Pdf link:** https://arxiv.org/pdf/2402.18416
 - **Abstract**
 We present the results of a preliminary study of a correction method applied to the Imaging Atmospheric Cherenkov Telescope images affected by clouds. The studied data are Monte Carlo simulations made with CORSIKA, imitating the very high energy events registered by the Large-Sized Telescopes, a type of telescope within the future Cherenkov Telescope Array. We implement the cloud correction method in the ctapipe/lstchain analysis framework. The correction is based on a simple geometrical model of the emission. We show the effect of the correction method on the image parameters and the stereo-reconstructed shower parameters.
#### On the Age Calibration of Open Clusters using Red Clump Stars
 - **Authors:** Abigail R. Chriss, Guy Worthey
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2402.18538
 - **Pdf link:** https://arxiv.org/pdf/2402.18538
 - **Abstract**
 In this study, we extend the dust-independent Hatzidimitriou (1991) relation between cluster age and $d_{B-R}$ color difference between red giant branch (RGB) and red clump (RC) to younger cluster ages. We perform membership analysis on twenty-two open clusters using $\textit{Gaia}$ DR3 astrometry, then compute the difference in color of red giant branch and red clump $d_{B-R}$ using $\textit{Gaia}$ photometry. We find that the trend derived from older clusters does not extrapolate to younger ages and becomes double-valued. We confirm that $d_{B-R}$ is independent of metallicity. Current stellar evolutionary isochrones do not quantitatively reproduce the trend and furthermore predict an increased color gap with a decrease in metallicity that is not echoed in the data. Integrated light models based on current isochrones exaggerate the color change over the $-0.5 <$ [Fe/H] $< 0$ interval at the few-percent level.

by olozhika (Xing Yuchen). 

2024-03-01
